recipes/geoindex_rs/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: geoindex_rs
```

This should probably be a `-` hyphen
```suggestion
  name: geoindex-rs
```